MungPlex/resources/settings.json

```diff
@@ -392,9 +392,9 @@
         ]
       },
       "DefaultWindowSelect": 0,
-      "DocumentsPath": "C:\\Users\\s_sch\\Documents",
-      "EnableRichPresence": true,
-      "Scale": 0.8941890001296997
+      "DocumentsPath": "C:\\Users\\Tim\\Documents",
```

Comment:
Maybe this file shouldn't be tracked by git either if it contains personalized settings and file paths.